apps/stripe/src/modules/saleor/saleor-money.test.ts

```diff
@@ -0,0 +1,81 @@
+import { describe, expect, it } from "vitest";
+
+import { SaleorMoney } from "./saleor-money";
+
+describe("SaleorMoney", () => {
+  describe("createFromStripe", () => {
+    it("creates a valid money object for valid input", () => {
+      const result = SaleorMoney.createFromStripe({ amount: 1000, currency: "usd" });
+
+      expect(result.isOk()).toBe(true);
+      expect(result._unsafeUnwrap()).toBeInstanceOf(SaleorMoney);
+    });
+
+    it("rejects negative amounts", () => {
+      const result = SaleorMoney.createFromStripe({ amount: -100, currency: "usd" });
+
+      expect(result.isErr()).toBe(true);
+      expect(result._unsafeUnwrapErr()).toBeInstanceOf(SaleorMoney.ValdationError);
+    });
+
+    it("rejects invalid currency code length", () => {
+      const result = SaleorMoney.createFromStripe({ amount: 100, currency: "usdd" });
+
+      expect(result.isErr()).toBe(true);
+      expect(result._unsafeUnwrapErr()).toBeInstanceOf(SaleorMoney.ValdationError);
+    });
+
+    it("rejects unsupported currency codes", () => {
+      const result = SaleorMoney.createFromStripe({ amount: 100, currency: "abc" });
+
+      expect(result.isErr()).toBe(true);
+      expect(result._unsafeUnwrapErr()).toBeInstanceOf(SaleorMoney.ValdationError);
+    });
+  });
+
+  describe("getAmount", () => {
```

Comment:
hint - you can use test.each for these, they are pretty easy cases (but this is fine too ofc)